### DISCUSSION TITLE
Welcome to mxcubecore Discussions!

### DISCUSSION THREAD

**Discussion by IvarsKarpics** [mxcube/mxcubecore#617] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marcus-oscarsson** [comment]
Great :)

**Reply by IvarsKarpics** [reply]
I never used the discussions in the git and will have a look how it works. Maybe it fits to our needs.

**Reply by marcus-oscarsson** [reply]
Me neither its a first, yes lets see if it does the job. I guess a first attempt might be to try to discuss the MEP idea (#614) is there an easy way to link a discussion to a PR in this case (#615) then ?

**Reply by marcus-oscarsson** [reply]
I created #618 but there does not seem to be an automatic way to link PRs and issues from the discussion. Which also makes me wonder why the discussions should just not take place directly in the PR of the MEP in question.

I guess this is more for discussions that are not directly related to a PR or issue then
